examples/free_tier_cluster/create_free_tier_cluster.tf

```
@@ -0,0 +1,20 @@
+
+output "new_free_tier_cluster" {
+  value = couchbase-capella_cluster_free_tier.new_cluster_free_tier
+}
+
+output "cluster_free_tierid" {
+  value = couchbase-capella_cluster_free_tier.new_cluster_free_tier.id
+}
+
+resource "couchbase-capella_cluster_free_tier" "new_cluster_free_tier" {
```

Can we do it like this instead? It will be more uniform everywhere
```suggestion
resource "couchbase-capella_free_tier_cluster" "new_free_tier_cluster" {
```

---

modified https://github.com/couchbasecloud/terraform-provider-couchbase-capella/pull/264/commits/65bc44126f5c74917552b4616f07d93a18152a4b